.changeset/rotten-buckets-smell.md

```diff
@@ -0,0 +1,5 @@
+---
+'@urql/storybook-addon': patch
+---
+
+- Add `@urql/devtools` to `@urql/storybook-addon`. Previously it was not included (See [#1092](https://github.com/FormidableLabs/urql/issues/1092)), but now the exchange works out of the box with Storybook, by [@r281gq](https://github.com/r281GQ)
```

Comment:
Quick note again 😅 sorry... the changeset changelog generator will add the attribution ("by ...") automatically once it generates the changelog updates

Reply:
No problemo! 👍  Corrected it.